diff --git a/node_modules/vitepress/dist/node/serve-3dd2c315.js b/node_modules/vitepress/dist/node/serve-3dd2c315.js
index c939f17..969eecc 100644
--- a/node_modules/vitepress/dist/node/serve-3dd2c315.js
+++ b/node_modules/vitepress/dist/node/serve-3dd2c315.js
@@ -35640,8 +35640,10 @@ function findRegion(lines, regionName) {
     // Visual Basic
     /^::#((?:end)region) ([\w*-]+)$/,
     // Bat
-    /^# ?((?:end)?region) ([\w*-]+)$/
+    /^# ?((?:end)?region) ([\w*-]+)$/,
     // C#, PHP, Powershell, Python, perl & misc
+    /^\s*\{\/\* #?((?:end)?region) ([\w*-]+) \*\/\}/,
+    // react tsx
   ];
   let regexp = null;
   let start = -1;
